drizzlepac/alignimages.py

```diff
@@ -331,7 +331,12 @@ def run_align(input_list, archive=False, clobber=False, debug=False, update_hdr_
     best_fit_rms = -99999.0
     best_fitStatusDict={}
     best_fitQual = 5
-    fit_algorithm_list= [match_2dhist_fit,match_default_fit]
+    # The match_relative_fit algorithm must have more than one image as the first image is
+    # the reference for the remaining images.
+    if len(imglist) > 1:
+        fit_algorithm_list=[match_relative_fit, match_2dhist_fit,match_default_fit]
+    else:
+        fit_algorithm_list=[match_2dhist_fit,match_default_fit]
     for catalogIndex in range(0, len(catalogList)): #loop over astrometric catalog
```

Comment:
You're are not recording the index value, or using it outside of access to the parent list, so you could simplify this:

```python
for catalog in catalogList:
    log.info("Astrometric Catalog: %s",str(catalog))
```

If you ever plan to record the index some day:

```python
for catalogIndex, catalog in enumerate(catalogList):
        log.info("Astrometric Catalog: {:02d}: {:s}".format(catalogIndex, str(catalog)))
```

Reply:
Deferred to Git Issue #295.